api/events/schema.py

```diff
@@ -254,4 +270,23 @@ def resolve_all_events(self, info, **kwargs):
                 )
             )
             all_days_events.sort(key=lambda x: datetime.strptime(x.day, "%a %b %d %Y"), reverse=True) # noqa
-        return all_days_events
+        if page and per_page:
```

Comment:
Good work @rajeman. Perhaps we could also have validations to ensure that whenever `page` is provided as an argument then `per_page` should also be provided so as not to have null results

Reply:
Nice idea. I will implement it.